docs/moja_global_docs/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+

```

Comment:
it is always good practice to write a short note after a topic or subtopic

Reply:
Okay, thank you for the suggestion, I will keep this in mind